services/QuillLMS/client/app/bundles/Evidence/components/studentView/promptStep.tsx

```diff
@@ -310,8 +319,19 @@ export class PromptStep extends React.Component<PromptStepProps, PromptStepState
     />)
   }
 
+  renderCharacterLimitWarning = (characterCount: number, characterCountClassName: string) => {
+    if(characterCount < 160) { return }
+    const message = characterCount < 200 ? APPROACHING_LIMIT_MESSAGE : OVER_LIMIT_MESSSAGE;
```

Comment:
We use both 160 and 200 in here a lot - might want to turn those into constants at the top of the file.